ci-operator/step-registry/cucushift/installer/check/vsphere/zone/rhcos-template/cucushift-installer-check-vsphere-zone-rhcos-template-commands.sh

```diff
@@ -43,7 +48,14 @@ function check_rhcos() {
 	fi
 
     done
-
+    if [[ $(yq-go r ${INSTALL_CONFIG} "platform.vsphere.failureDomains[*].topology.template" | wc -l) == "${#zones_name_from_config[@]}" ]];then
+        if grep "Obtaining RHCOS image file" "${dir}"/.openshift_install.log; then
```

Comment:
I don't think openshift-install.log exists under folder /tmp/installer in this step, it is only available in step `ipi-install-install`, but seems that log file is also not copied into ${SHARED_DIR} there, might need to copy into ${SHARED_DIR} firstly, then use it here.

Reply:
I saw the install log file was copied to `${ARTIFACT_DIR}/.openshift_install-${current_time}.log`

Reply:
yes, but not copied to ${SHARED_DIR}, if we want to use it in later step, maybe need to update  https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L234-L238

Reply:
Ah, you are right, I forgot `${SHARED_DIR}` is not shared, either.

Reply:
```suggestion
        if grep "Obtaining RHCOS image file" "${SHARED_DIR}"/.openshift_install.log; then
```